### PR TITLE
Possible integrations

### PR DESCRIPTION
**Pull request by gallegonovato**
First of all, please excuse me. I don't know what's wrong with my internet provider today, it's going in jumps. I called my operator and it is their fault.

I put some connections that I found, but I do not know if some are already repeated, since I had them noted in the mobile (from which I am) and I have no possibility to see if they are or not repeated. If they are I beg your pardon and I also beg you not to kill me jejej

I put things from duck, sd maid, or mozilla that create weird connections.

A hug and I apologize for bothering you so much and more in these dates.

### CODE REVIEW

File: pihole-google.txt [hunk 1]
```diff
@@ -7719,3 +7719,6 @@ r3---sn-h0jeln7l.gvt.com
 r3.sn-h0jeln7l.gvt.com
 r5---sn-h0jeen7d.gvt.com
 r5.sn-h0jeen7d.gvt.com
+s.youtube.com
+#Amazon
+acsechocaptiveportal.com
```
Comment (nickspaargaren) [inline pihole-google.txt:7724]: Amazon domains should be removed.

You can make pull request for Amazon in https://github.com/nickspaargaren/no-amazon

File: pihole-google.txt [hunk 2]
```diff
@@ -7399,4 +7404,321 @@ www.tf-enterprise-devel.gcp.fitbit.com
 www.vps.store-au.fitbit.com
 www.wiki.fitbit.com
 www.windchill.fitbit.com
-wwwinternal.fitbit.com
\ No newline at end of file
+wwwinternal.fitbit.com
+android-safebrowsing.googleapis.com
+developers.google.com
+malware-error.mozilla.con
+phish-error.mozilla.con
```
Comment (nickspaargaren) [inline pihole-google.txt:7411]: mozilla.con domains do not exist and are not linked to Google. These should be removed

File: pihole-google.txt [hunk 3]
```diff
@@ -7399,4 +7404,321 @@ www.tf-enterprise-devel.gcp.fitbit.com
...
+bugsnag.com
+crashlytics.com
+e.crashlytics.com
+firebaseinstallations.googleapis.com
+firebase-settings.crashlytics.com
+graph.facebook.com
+piwik.darken.eu # SD Maid
+privatestats.whatsapp.net # Most important WhatsApp tracker
+reports.crashlytics.com
+sessions.bugsnag.com
```
Comment (nickspaargaren) [inline pihole-google.txt:7432]: bugsnag.com domains do not exist and are not linked to Google. These should be removed

File: pihole-google.txt [hunk 4]
```diff
@@ -7399,4 +7404,321 @@ www.tf-enterprise-devel.gcp.fitbit.com
...
+secureframe.doubleclick.net
+securepubads.g.doubleclick.net
+static.doubleclick.netnod
+stats.g.doubleclick.net
+survey.g.doubleclick.net
+## Google-analytics.com
+google-analytics.com
+ssl.google-analytics.com
+www.google-analytics.com
+## Google.com
```
Comment (nickspaargaren) [inline pihole-google.txt:7502]: Please remove all extra comments

File: pihole-google.txt [hunk 5]
```diff
@@ -7399,4 +7404,321 @@ www.tf-enterprise-devel.gcp.fitbit.com
...
+android.googleapis.com
+playatoms-pa.googleapis.com
+play.googleapis.com
+android.clients.google.com
+cloudconfig.googleapis.com
+app-measurement.com
+ssl.google-analytics.com
+connectivitycheck.android.com
+example.org
+ipv4only.arpa
```
Comment (nickspaargaren) [inline pihole-google.txt:7602]: ipv4only.arpa domains are not linked to Google. These should be removed

File: pihole-google.txt [hunk 6]
```diff
@@ -7399,4 +7404,321 @@ www.tf-enterprise-devel.gcp.fitbit.com
...
+phish-error.mozilla.con
+safebrowsing.google.com
+safebrowsing.googleapis.com
+sb-ssl.google.com
+sb.l.google.com
+acceleratedmobilepageurl.googleapis.com
+android.bugly.qq.com
+app-measurement.com
+#Duckduckgo
+improving.duckduckgo.com
```
Comment (nickspaargaren) [inline pihole-google.txt:7420]: Duckduckgo.com domains are not linked to Google. These should be removed

File: pihole-google.txt (hunk 2 above)
Comment (gallegonovato) [reply pihole-google.txt]: All changes made

File: pihole-google.txt (hunk 6 above)
Comment (gallegonovato) [reply pihole-google.txt]: All changes made

File: pihole-google.txt (hunk 3 above)
Comment (gallegonovato) [reply pihole-google.txt]: All changes made

File: pihole-google.txt (hunk 5 above)
Comment (gallegonovato) [reply pihole-google.txt]: All changes made

File: pihole-google.txt (hunk 1 above)
Comment (gallegonovato) [reply pihole-google.txt]: Removed and I will put it in the one that tells me and if I see more from Amazon?

Comment (nickspaargaren) [reply pihole-google.txt]: Thanks for your adjustments! Ans sure, add as many as you can to https://github.com/nickspaargaren/no-amazon !

File: pihole-google.txt [hunk 7]
```diff
@@ -7399,4 +7405,293 @@ www.tf-enterprise-devel.gcp.fitbit.com
...
+developers.google.com
+safebrowsing.google.com
+safebrowsing.googleapis.com
+sb-ssl.google.com
+sb.l.google.com
+acceleratedmobilepageurl.googleapis.com
+android.bugly.qq.com
+app-measurement.com
+autofill.googleapis.com
+bugly.qq.com
```
Comment (nickspaargaren) [inline pihole-google.txt:7419]: qq.com domains are not linked to Google. These should be removed

File: pihole-google.txt [hunk 8]
```diff
@@ -7399,4 +7405,293 @@ www.tf-enterprise-devel.gcp.fitbit.com
...
+clients3.google.com
+clients4.google.com
+connectivitycheck.gstatic.com
+mobile-gtalk.l.google.com
+play-lh.googleusercontent.com
+sb.l.google.com
+supl.google.com
+redirector.gvt1.com
+beacons.gvt2.com
+beacons3.gvt2.com
```
Comment (nickspaargaren) [inline pihole-google.txt:7691]: gvt.com domains are not linked to Google. These should be removed

File: pihole-google.txt (hunk 7 above)
Comment (gallegonovato) [reply pihole-google.txt]: Done

File: pihole-google.txt (hunk 8 above)
Comment (gallegonovato) [reply pihole-google.txt]: Should I delete all of them from gvt?

Comment (nickspaargaren) [reply pihole-google.txt]: Whoops, `gvt1.com` and `gvt2.com` are indeed Google's. Only `gvt.com` is not. It's probably best if you revert the last commit.

Comment (gallegonovato) [reply pihole-google.txt]: como podria hacer eso ?. no me permite cargar las cosas.

Comment (gallegonovato) [reply pihole-google.txt:7656]: now I think they are already on

**Comment by nickspaargaren** [comment]
Merged! You can see my adjustments [here](https://github.com/nickspaargaren/no-google/pull/100). I've added a script so we get a red cross if a pull request has duplicate domains. This is why I needed to remove some domains first. Thanks again @gallegonovato!

**Comment by gallegonovato** [comment]
Good afternoon @nickspaargaren

 No problem. I'm glad I was of help to you. 

The only thing I do not know if you saw the hosts that I put in https://github.com/nickspaargaren/no-google/issues/91 

There I have put everything to avoid Google and connections that I was finding. 

Now if you want I look at things from Microsoft or Amazon. 

MERRY CHRISTMAS AND HAPPY NEW YEAR

**Pull request merged**